### ISSUE TITLE
[Bug]: ListBox not Accessible with VoiceOver on Mac

### ISSUE THREAD

**Issue by chigkim** [juce-framework/JUCE#1204]
### Detailed steps on how to reproduce the bug

Build and run MidiDemo and navigate input/output device list.

### What is the expected behaviour?

VoiceOver should announce device name.

### Operating systems

macOS

### What versions of the operating systems?

12.6.3

### Architectures

x86_64, 64-bit

### Testing on the `develop` branch

The bug is present on the `develop` branch

### Code of Conduct

- [X] I agree to follow the Code of Conduct
